disnake/client.py

```diff
@@ -97,7 +98,13 @@
     "GatewayParams",
 )
 
+T = TypeVar("T")
+
+Coro = Coroutine[Any, Any, T]
+CoroFunc = Callable[..., Coro[Any]]
+
 CoroT = TypeVar("CoroT", bound=Callable[..., Coroutine[Any, Any, Any]])
```

Comment:
```suggestion
```
nit: this is now equivalent to `CFT`, so we can remove one of them and replace occurrences with the other one